lib/idl/idl_checkers.es6.js

```diff
@@ -65,5 +65,21 @@ module.exports = [
     logger.info('Checking top-level member names');
 
     eachKey(logger, left.members || [], right.members || [], 'name');
-  }
+  },
+  function rightMissingConstructor(logger, left, right) {
```

Comment:
It seems to me that right now it only works for the case when spec has a constructor and blink doesn't.
Philip mentioned that an interface may has multiple constructors. Is it possible to count the number of constructors of left and right, and do a diff check?